velox/exec/tests/TableScanTest.cpp

```diff
@@ -238,6 +238,27 @@ TEST_F(TableScanTest, columnAliases) {
   assertQuery(op, {filePath}, "SELECT c0 FROM tmp WHERE c0 % 2 = 1");
 }
 
+TEST_F(TableScanTest, decimal) {
```

Comment:
Have checked out tests in this file velox/dwio/parquet/tests/duckdb_reader/ParquetTableScanTest.cpp.

You can follow similar approach for decimals. Create a parquet file externally using python script.

```
// With filters
  assertSelectWithFilter(
      {"a"},
      common::test::singleSubfieldFilter("a", exec::lessThan(3)),
      "SELECT a FROM tmp WHERE a < 3");
```
There are tests on subfield filters here. May be these tests belong here.

Reply:
Thanks for the suggestion, looking into tests here.